pool/app/pool_common/entity_i18n.ml

```diff
@@ -124,6 +124,8 @@ type hint =
   | LocationFiles
   | Locations
   | LocationSessions
+  | MissingMessageTemplates of string * string list (* TODO[timhub]*)
+  (* | MissingMessageTemplates of string * Entity.Language.t list *)
```

Comment:
@aronerben I forgot about this todo, but maybe you can help me 😄 

If I replace the `string list` with `Entity.Language.t list`, the compiler complains: 

```
Type Pool_common.Language.t is not compatible with type
         Pool_common__Entity.Language.t 
```

Any idea how to solve that?

Reply:
We can leave it as that for now (checked in person)